pkgs/development/python-modules/livekit-protocol/default.nix

```diff
@@ -0,0 +1,41 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  protobuf,
+  types-protobuf,
+}:
+
+buildPythonPackage rec {
+  pname = "livekit-protocol";
+  version = "1.0.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "livekit";
+    repo = "python-sdks";
+    tag = "protocol-v${version}";
+    hash = "sha256-opYF9BmZ8reuyo6IeBjdJC6YQGypnWGHLSY3phJ4cxQ=";
+    fetchSubmodules = true;
```

Comment:
The code does not look like it requires the gitmodules for this package.

```suggestion
```

and invalidate the hash.